Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Templates/FieldsAndConstructor.tt

```diff
@@ -8,18 +8,32 @@
     if (_model.LambdaMethod.UsingDependencyInjection)
     {
 #>
-        private readonly ServiceProvider serviceProvider;
+        private <#= _model.IsExecutable ? "static " : "" #>readonly ServiceProvider serviceProvider;
 <#
     }
     else
     {
 #>
-        private readonly <#= _model.LambdaMethod.ContainingType.Name #> <#= _model.LambdaMethod.ContainingType.Name.ToCamelCase() #>;
+        private <#= _model.IsExecutable ? "static " : "" #>readonly <#= _model.LambdaMethod.ContainingType.Name #> <#= _model.LambdaMethod.ContainingType.Name.ToCamelCase() #>;
```

Comment:
The ternary operator for static should be removed since we aligned on the wrapping generated class always being an instance class.